extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -9,6 +9,7 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 		'DirectGptMobile',
 		'mobile',
 		{
+			INVISIBLE_HIGH_IMPACT:      {size: '1x1'},
 			MOBILE_TOP_LEADERBOARD:     {size: '320x50,320x100,1x1'},
```

Comment:
I'm wondering if I should now remove `1x1` from all other slots. What do you think?